### DISCUSSION TITLE
How does it work?

### DISCUSSION THREAD

**Discussion by Inedhelplol** [marioparaschiv/nitro-sniper#113] [General]
So I've just checked the Application logs and it says aready redeemed does this mean someone already used it? 
btw is this an auto snipe so it will automatically send it to my discord webhook?

**Comment by MrD4rio** [comment]
When it says already redeemed it's most likely an already redeemed code, or someone else's sniper was faster in redeeming.
And yeah, if you input the webhook link in the .env file the snipes will be sent to your own webhook.
